### PR TITLE
Updated the Readme Doc

### PR DESCRIPTION
**Pull request by Devbysteph**
I improved the Readme doc, please review it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
+>When you send someone a secret message, our platform protects your privacy to keep you anonymous. You remain anonymous unless you decide to expose your true identity.
+2. Simple to Use 💫
+>We are continually improving the platform, to make AnonMsg🙈 as user-friendly as possible.To get started right now, all you need to do is to go [AnonMsg🙈](https://anon-msg-app.vercel.app/)and enter your login and password. 
+3. Secure & Private 🔒
+>With AnonMsg🙈 messages are secured with end to end encryption and your account is secure.
+4. 100% Free 🆓
+>Access all of AnonMsg🙈 features for free without paying a penny.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+## Download
```
Comment (vicradon) [inline README.md]: Might want to replace this download with "View". You know it's a web app.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+## App Screenshots
```
Comment (vicradon) [inline README.md]: I think we can add this now. Both the desktop and mobile version.

File: README.md [hunk 3]
```diff
@@ -1,34 +1,29 @@
...
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+In order for users' connections and followers to participate in anonymous messaging, AnonMsg makes it incredibly simple for users to post their link on other social networking sites as well.
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+You may check out how it works and [get started](https://anon-msg-app.vercel.app/) for free if you don't already have an account.
+## Features
+1. Anonymity 👓
+>When you send someone a secret message, our platform protects your privacy to keep you anonymous. You remain anonymous unless you decide to expose your true identity.
+2. Simple to Use 💫
+>We are continually improving the platform, to make AnonMsg🙈 as user-friendly as possible.To get started right now, all you need to do is to go [AnonMsg🙈](https://anon-msg-app.vercel.app/)and enter your login and password.
```
Comment (vicradon) [inline README.md]: Might want to replace the phrase "end to end" encryption with: "aes cipher".

File: README.md [hunk 4]
```diff
@@ -1,34 +1,29 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# AnonMsg🙈
 
-## Getting Started
+<img src="./public/Emoji.png" style: width="100px">
```
Comment (vicradon) [inline README.md]: This image is broken... I think it has to do with the relative link. Try removing the `./public`.

**Review by vicradon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Devbysteph) [reply README.md]: Okayy sure

File: README.md (hunk 2 above)
Comment (Devbysteph) [reply README.md]: Alright, noted. Is it responsive now?

File: README.md (hunk 4 above)
Comment (Devbysteph) [reply README.md]: Alright, the saved the image on the public folder that was why I referenced it. What do you think. I will look into you ASAP.

File: README.md (hunk 3 above)
Comment (Devbysteph) [reply README.md]: Okay, Roger that.

File: README.md [hunk 5]
```diff
@@ -1,34 +1,37 @@
...
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+# AnonMsg🙈
+![Logoplaceholder](https://github.com/Devbysteph/anon-msg-app/blob/main/public/Logo%20placeholder.png)
```
Comment (vicradon) [inline README.md]: Can you also structure this image like the others? If it references your repo, it might not show if your repo is moved.

**Review by vicradon — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Devbysteph) [reply README.md]: I've made modifications to them, thank you for pointing it out.

**Review by vicradon — APPROVED** [review]

**Comment by vicradon** [comment]
Thanksyou @Devbysteph for this contribution.

**Pull request merged**